plugin.video.ispot/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,26 @@
+# Kodi Media Center language file
+# Addon Name: ispot
+# Addon id: plugin.video.ispot
+# Addon Provider: Lunatixz
+msgid ""
+msgstr ""
+"Project-Id-Version: plugin.video.ispot\n"
+"Report-Msgid-Bugs-To: http://forum.kodi.tv/showthread.php?tid=311057\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Lunatixz Translation Team\n"
+"Language-Team: English (http://www.transifex.com/projects/p/xbmc-addons/language/en/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgctxt "#42001"
```

Comment:
please use the correct range of id's: strings 30000 thru 30999 reserved for plugins and plugin settings
(see: http://kodi.wiki/view/language_support)